src/android/SharedPreferencesHandler.java

```diff
@@ -14,6 +14,11 @@ public SharedPreferencesHandler (String prefsName, Context ctx){
 		prefs = ctx.getSharedPreferences(prefsName  + "_SS", 0);
 	}
 
+	boolean isEmpty() {
+	    int numOfPrefs = prefs.getAll().size();
```

Comment:
There is no need to store the result in a variable and then immediately return it. Just return it directly. Besides that `.isEmpty()` should be used instead of `.size() == 0`